src/main/java/uk/gov/companieshouse/api/accounts/model/entity/notes/creditorsafteroneyearentity/CurrentPeriodEntity.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.companieshouse.api.accounts.model.entity.notes.creditorsafteroneyearentity;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class CurrentPeriodEntity {
+
+    @Field("bank_loans_and_overdrafts")
```

Comment:
I imagine this will be the same for all notes with a current and previous period as they generally contain the same fields.

Reply:
Marco's correct, we could have a single model with these fields. Then in the parent model we'd include the resource twice, one annotated with `@Field("current_period")` and once with `@Field("previous_period")`

Reply:
Due to time constraints and consistency with notes that have already been developed, I think it would be best to keep the models as they are, and set a story for a refactoring for a future date.